R/access.R

```diff
@@ -0,0 +1,43 @@
+#' Check access to lookups
+#'
+#' @description
+#' `have_access()` provides a convenient way to check that you have the
+#' correct access to the necessary folders.
+#'
+#' @returns
+#' `[logical(1)]` - `TRUE` if you have appropriate access and `FALSE` otherwise.
+#'
+#' @examples
+#' \dontrun{
+#' library(phslookups)
+#' if (have_access()) {
+#'   print("Success!")
+#' } else {
+#'   print("Maybe next time")
+#' }
+#' }
+#' @export
+have_access <- function() {
```

Comment:
I'm not sure we want to / need to export it. Either way, a different name might be better - `check_lookups_access` or `check_unix_acute_access` would be my suggestions...

I think it could probably also be combined with `read_access_to_dir` unless you can think of a reason it's useful to have these two as separate functions?

Reply:
Have made the suggested updates. I also took away the message on successful access because users probably don't want to see that when running a script as it's not entirely useful or clear.